source/crud/tests/unified/estimatedDocumentCount.yml

```diff
@@ -5,6 +5,7 @@ schemaVersion: "1.0"
 createEntities:
   - client:
       id: &client0 client0
+      useMultipleMongoses: false # Avoid setting fail points with multiple mongoses
```

Comment:
Sorry for not catching this in #905. FWIW, this is already advised in the Unified Test Format spec under [failPoint](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#failpoint).